recommendation/dlrm/pytorch/docker_gpu/Dockerfile

```diff
@@ -38,28 +38,20 @@ RUN pip install --upgrade pip && \
     ldconfig
 
 RUN conda install pytorch torchvision -c pytorch
-RUN pip install tensorflow-gpu==1.14 onnxruntime-gpu
```

Comment:
Does this reference code depend on tensorflow packages (since this a reference implementation for pytorch)?